pkgs/development/python-modules/pyairvisual/default.nix

```diff
@@ -24,7 +24,7 @@ buildPythonPackage rec {
     owner = "bachya";
     repo = pname;
     rev = version;
-    sha256 = "0z769xrb6w6bhqcq02sjryl1qyvk9dc1xfn06fc3mdqnrbr0xxj3";
+    sha256 = "0jjvng3py5g97gvx6rdbk5zxbn5rw8gq1ki4qi4vfsypchxbpz2q";
   };
 
   nativeBuildInputs = [ poetry ];
```

Comment:
Please create an upstream issue to migrate the build-backend to `poetry-core`. See https://github.com/NixOS/nixpkgs/issues/103325.

Reply:
Done

https://github.com/bachya/pyairvisual/issues/69